src/pages/index.js

```diff
@@ -21,103 +22,278 @@ import {
   modalNameInput,
   modalDescriptionInput,
   addNewCardButton,
+  avatarEditButton,
+  avatarSelector,
+  modalChangeProfileAvatarSelector,
+  editModalFormSelector,
+  addCardFormSelector,
+  avatarModalFormSelector,
+  cardModalDelete,
 } from "../utils/constants.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 
-/**************
- * VALIDATION *
- **************/
-const addFormValidator = new FormValidator(
-  validationSettings,
-  addCardEditForm
-);
-addFormValidator.enableValidation();
-
-const editFormValidator = new FormValidator(
-  validationSettings,
-  profileEditForm
-);
-
-editFormValidator.enableValidation();
+const api = new Api({
+  baseUrl: "https://around.nomoreparties.co/v1/group-12",
+  headers: {
+    authorization: "9c860865-e6d3-4014-b437-60037dde85fb",
+    "Content-Type" : "application/json",
+  },
+});
+  
 
 /*************
- * CARD INFO *
+ * USER INFO *
  *************/
+const userInfo = new UserInfo({
+  userNameSelector,
+  userDescriptionSelector,
+  avatarSelector,
+});
 
-const userInfo = new UserInfo({ userNameSelector, userDescriptionSelector });
+let userId, cardSection;
+
+function renderCard(cardData) {
+  const cardImage = createCard(cardData);
+  cardSection.prependItem(cardImage);
+}
+
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([userData, cardData]) => {
+    userInfo.setUserInfo({
+      title: userData.name,
+      description: userData.about,
+      avatar: userData.avatar,
+    });
+    // userInfo.setAvatarInfo(userData.avatar);
+    userId = userData._id;
+    cardSection = new Section(
+      {
+        data: cardData,
+        renderer: renderCard,
+      },
+      cardListSelector
+    );
+    cardSection.renderItems();
+  })
+  .catch((err) => {
+    console.error(err);
+  });
 
 export const cardTemplate = document
   .querySelector("#card-template")
   .content.querySelector(".card");
 
+/********************
+ * POPUP WITH IMAGE *
+ ********************/
 const modalWithImage = new PopupWithImage({
-  popupSelector: imageModalSelector,
+  modalSelector: imageModalSelector,
 });
 
 const modalWithFormUser = new PopupWithForm({
```

Comment:
Variable name should describe variable value. `avatarModal` would be good

Reply:
.

Reply:
Variable name should describe variable value. For example `avatarModal` would be the good name for this PopupWithForm instance

Reply:
.

Reply:
Don't forget to fix this